packages/documentation-framework/components/example/example.js

```diff
@@ -105,7 +105,8 @@ export const Example = ({
     // These 2 are in the bundle anyways for the site since we dogfood
     ...reactCoreModule,
     ...reactTableModule,
-    ...(source === 'react-next' ? reactCoreNextModule : {})
+    ...(source === 'react-next' ? reactCoreNextModule : {}),
+    ...(source === 'react-deprecated' ? reactCoreDeprecatedModule : {})
```

Comment:
It doesn't look like `reactCoreDeprecatedModule` is defined, and I think we want to extend upon `react-table`'s module for this scope and not `react-core` (I assume we'll want both eventually, but for [this change](https://github.com/patternfly/patternfly-react/pull/8362), at least `react-table`).

Reply:
truee